helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/TestContext.java

```diff
@@ -0,0 +1,51 @@
+package org.symphonyoss.symphony.bots.helpdesk.bot.it;
+
+import org.symphonyoss.client.model.Room;
+import org.symphonyoss.symphony.clients.model.SymUser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by crepache on 22/02/18.
+ * Class to store the variables to use on any part of tests.
+ */
+public class TestContext {
+
+  private static TestContext instance;
```

Comment:
You should perform eager initialization and then remove the synchronized directive from the getInstance method.

```
private static final TestContext INSTANCE = new TestContext();
```

Reply:
Done.